tests/unit/helpers/questionnaires.test.js

```diff
@@ -284,54 +330,168 @@ describe('getUserQuestionnaires', () => {
     );
   });
 
-  it('should return an empty array if no questionnaire', async () => {
+  it('should return an empty array if course has slots to plan', async () => {
```

Comment:
is started and has slotss to plan

Reply:
is ended plutôt

Reply:
du coup non elle n'est pas terminée car elle a des creneaux a planifié